### DISCUSSION TITLE
Maintenance warning

### DISCUSSION THREAD

**Discussion by gumbarros** [ncalc/ncalc#307] [General]
@Bykiev I don't have your email, so I'm leaving a message here.

I'm going to be out for 20 days, feel free to merge any PRs and/or publish new versions when needed.

**Comment by Bykiev** [comment]
Thanks, I'll try not to break it :)

**Reply by gumbarros** [reply]
Thank you very much for your help, I'm back at home today :)

**Reply by Bykiev** [reply]
Welcome back!
